pkg/build/builder/dockerutil.go

```diff
@@ -62,15 +81,27 @@ func pushImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	if glog.V(5) {
 		opts.OutputStream = os.Stderr
 	}
-	var err error
+
 	var retriableError = false
 
+	eventWatch, err := watchForFailedPushEvent(eventClient, ref.Namespace, ref.Name)
```

Comment:
No, the carry patch should not be for here, and the daemon doesn't
propagate.  That problem doesn't have to be fixed here.

On Thu, Apr 7, 2016 at 12:23 PM, Michal Minar <notifications@github.com>
wrote:

> In pkg/build/builder/dockerutil.go
> <https://github.com/openshift/origin/pull/8195#discussion_r58901151>:
>
> >  	var retriableError = false
> >
> > +	eventWatch, err := watchForFailedPushEvent(eventClient, ref.Namespace, ref.Name)
>
> But I'd rather avoid adding the carry patch here. Also docker daemon
> doesn't propagate error details in this case. And I'm reluctant to put the
> details into error message
> <https://github.com/miminar/distribution/blob/master/registry/api/errcode/register.go#L30>
> instead of the description. This will need some more thought.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8195/files/182833487b48f39d14835bb857f60d2f9aad8099#r58901151>
>